slack/incident_commands.py

```diff
@@ -61,3 +63,27 @@ def set_severity(incident: Incident, user_id: str, message: str):
     comms_channel.post_in_channel(f"The incident has been running for {duration}")
 
     return True, None
+
+@incident_command(['close'], helptext='Close this incident.')
+def set_severity(incident: Incident, user_id: str, message: str):
+    comms_channel = CommsChannel.objects.get(incident=incident)
+
+    if incident.is_closed():
+        comms_channel.post_in_channel(f"The incident was already closed at {incident.end_time.strftime('%Y-%m-%d %H:%M:%S')}")
+        return True, None
+
+    incident.end_time = datetime.now()
+    incident.save()
+
+    comms_channel.post_in_channel(f"The incident has been closed! 📖 -> 📕")
+
+    return True, None
+
+@incident_command(['action'], helptext='Set an action for this event')
+def set_action(incident: Incident, user_id: str, message: str):
+    comms_channel = CommsChannel.objects.get(incident=incident)
+    if incident.lead:
+        Action(incident=incident, details=message, user=incident.lead).save()
+    else:
+        comms_channel.post_in_channel(f"Actions require an incident lead")
```

Comment:
Rather than refuse to create an action, which might introduce enough friction for someone not to bother, can we fall back to assigning the person creating the action as the owner?